GlyssenTests/BlockTests.cs

```diff
@@ -1375,28 +1375,113 @@ public void GetSwappedReferenceText_RowBIsNull_EntireContentsSwap()
 			Assert.AreEqual("{21} Verse twenty-one.", newRowBValue);
 		}
 
-		private CharacterVerse JesusQuestioning
+		[TestCase("1234567")]
```

Comment:
Should probably also have a couple tests for Length to cover the special case of ChapterAnnouncements.